README.md

```diff
@@ -17,32 +17,36 @@ The main use is for adding **cycles** and **persistence** to your LLM applicatio
 
 Cycles are important for agentic behaviors, where you call an LLM in a loop, asking it what action to take next.
 
+### Key Features
+
+- **Stateful Graphs**: Maintain state across nodes for complex, multi-step computations.
```

Comment:
what do we think of saying "fault-tolerant" or "error-tolerant" or something instead of "complex"?

Reply:
good question -- i am actually think it just needs to go in favor of `Persistence` below since it's duplicative